.bazelrc

```diff
@@ -1,12 +1,17 @@
 # Requires at least bazel 6
 common --enable_bzlmod
 
+build --enable_platform_specific_config
+
 # C++, with warnings mostly turned to 11
-build --cxxopt=-xc++                    --host_cxxopt=-xc++
-build --cxxopt=-std=c++23               --host_cxxopt=-std=c++23
-build --cxxopt=-Wall                    --host_cxxopt=-Wall
-build --cxxopt=-Wextra                  --host_cxxopt=-Wextra
-build --cxxopt=-Wno-unused-parameter    --host_cxxopt=-Wno-unused-parameter
+build:macos --macos_minimum_os=10.15
+build:macos --features=-supports_dynamic_linker
+build:macos --cxxopt=-std=c++2b                   --host_cxxopt=-std=c++2b
+build:linux --cxxopt=-std=c++23                   --host_cxxopt=-std=c++23
```

Comment:
This should be the standard for all operating systems _except_ the ones that have ancient compilers. Otherwise we now would need to enumerate also bsd, windows, ... Is it possible to declare that ?

(If not, don't worry, but we should at least research that. Would be a bummer just because Apple is not updating their operating systems that we have to live with such a wart)

Reply:
I searched a bit more, but I still couldn't find a way to enable -std=c++23 :(